clean-abap/CleanABAP.md

```diff
@@ -4606,6 +4606,15 @@ by redirecting them to test data defined in the unit test without interfering wi
 
 - Use the CDS test environment (`CL_CDS_TEST_ENVIRONMENT`) to test your CDS views.
 
+- Use the authorization double (`CL_AUNIT_AUTHORITY_CHECK`) to withdraw authorizations during the runtime of the test.
+  This helps to test behavior of the code for different authorizations.
+
+```ABAP
+    DATA(auth_controller) = cl_aunit_authority_check=>get_controller( ).
+    DATA(auth_obj_set) = cl_aunit_authority_check=>create_auth_object_set(  user_role_authorizations ).
```

Comment:
This conflicts with rule "Avoid abbreviations". I suggest using the variable names `authority_check_controller` and `authority_object_set` / `authorization_object_set`. 

I can understand wanting to abbreviate to `auth`, but I don't think we should abbreviate `objects`.

(Also, there's an unnecessary space before `user_role_authorizations`.)

Reply:
Valid point, I will look into this after my vacation.